messages/ar.json

```diff
@@ -1,60 +0,0 @@
-{
```

Comment:
Was this removal intentional? Haven't seen it mentioned anywhere.

Reply:
yes :D